content_script.js

```diff
@@ -1,32 +1,33 @@
-/** 
+/**
   * Class Schedule to ICS
-  * (c) 2013 Keanu Lee
+  * (c) 2013-2015 Keanu Lee
+  * (c) 2015 Baraa Hamodi
   *
   * Get ICS files for university class schedules in Oracle PeopleSoft systems (including UW Quest)
-  * 
+  *
   * TODO list:
   * - Properly handle date numbers (e.g. 20130901 - 1 != 20130900, 20130131 + 1 != 20130132)
 **/
 
 $(function() {
-
   // Ontario is UTC-4
+  // TODO: Create a getUTCOffset function that handles day light savings.
   utc_offset  = -4;
 
 
   // 05/17/1992 -> 19920517
   function getDateString(date) {
-    return date.substr(6,4)
-      + date.substr(0,2)
-      + date.substr(3,2);
+    return date.substr(6,4) +
+            date.substr(0,2) +
+            date.substr(3,2);
   }
 
   // 4:30PM -> 203000
   function getTimeString(time) {
-    time_string = (parseInt(time.replace(/[:APM]/g,""))
-        + (time.match(/P/) ? 1200 : 0)
-        - (time.match(/12:\d\dPM/) ? 1200 : 0)
-        - (utc_offset * 100)) * 100;
+    time_string = (parseInt(time.replace(/[:APM]/g, "")) +
+                  (time.match(/P/) ? 1200 : 0) -
```

Comment:
Indentation here is fine, since it aligns with the above lines. Do whichever you prefer, here and elsewhere.